### PR TITLE
cuSignal Deprecation Notice

### PR DESCRIPTION
**Pull request by caryr35**
cusignal repository is being deprecated and. is integrating into CuPy repository

### CODE REVIEW

File: _notices/rsn0032.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+notice_topic: Platform Support Change
+notice_rapids_version: "v23.08"
+notice_created: 2023-05-26
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2023-05-26
+---
+
+## Overview
+
+RAPIDS cuSignal repository is integrating into the CuPy repository and targeting deprecation in 3 to 4 months. We are tentativley targeting the v23.08 release scheduled for August 10, 2023. After the v23.08 release, this repository will be archived during v23.10 release. v23.08 is the targeted final release of RAPIDS cuSignal..
```
Comment (bdice) [inline _notices/rsn0032.md:29]: Re-ordered a couple sentences to make it flow better.
```suggestion
RAPIDS cuSignal repository is integrating into the CuPy repository and targeting deprecation in 3 to 4 months. We are tentatively targeting the v23.08 release scheduled for August 10, 2023. v23.08 is the targeted final release of RAPIDS cuSignal. After the v23.08 release, this repository will be archived during the v23.10 development cycle.
```

Comment (aravenel) [inline _notices/rsn0032.md:29]:
```suggestion
RAPIDS cuSignal is being integrated directly into CuPy. Accordingly, cuSignal is being deprecated and its repository will be archived. Specific date for archival is still TBD, but we estimate approx. 3-4 months until EOL, with 23.08 likely to be the final release of cuSignal.
```

File: _notices/rsn0032.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+
+## Impact
+Users should to migrate to CuPy, which includes the cuSignal functionality: https://github.com/cupy/cupy
```
Comment (aravenel) [inline _notices/rsn0032.md:33]:
```suggestion
Users should prepare to migrate to CuPy, which will include cuSignal functionality: https://github.com/cupy/cupy
```

Comment (bdice) [inline _notices/rsn0032.md:33]: I think we can clarify this is a future feature, not a current feature.
```suggestion
Users should to migrate to CuPy, which will include the cuSignal functionality: https://github.com/cupy/cupy
```

File: _notices/rsn0032.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 32 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Deprecation announcement for cuSignal RAPIDS Repository"
```
Comment (aravenel) [inline _notices/rsn0032.md:10]:
```suggestion
title: "Deprecation announcement for cuSignal"
```

File: _notices/rsn0032.md (hunk 1 above)
Comment (aravenel) [inline _notices/rsn0032.md:29]:
```suggestion
RAPIDS cuSignal repository is integrating into the CuPy repository and targeting deprecation in 3 to 4 months. We are tentativley targeting the v23.08 release scheduled for August 10, 2023. After the v23.08 release, this repository will be archived during v23.10 release. v23.08 is the targeted final release of RAPIDS cuSignal.
```

Comment (aravenel) [reply _notices/rsn0032.md:29]: I actually prefer Brad's text :)

Comment (bdice) [reply _notices/rsn0032.md:29]: Sorry! We overlapped in our commenting.

**Review by aravenel — APPROVED** [review]

**Pull request merged**